spec/services/receive_reference_spec.rb

```diff
@@ -4,8 +4,8 @@
   it 'updates the reference on an application form with the provided text' do
     application_form = FactoryBot.create(:completed_application_form, references_count: 0)
     application_form.application_choices.each { |choice| choice.update(status: 'awaiting_references') }
-    create(:reference, :unsubmitted, email_address: 'ab@c.com', application_form: application_form)
-    create(:reference, :unsubmitted, email_address: 'xy@z.com', application_form: application_form)
+    application_form.references << build(:reference, :unsubmitted, email_address: 'ab@c.com')
+    application_form.references << build(:reference, :unsubmitted, email_address: 'xy@z.com')
```

Comment:
`application_form: application_form` is removed here but kept in the  creates below.
Is this intentional?

Reply:
Done!